docker/resources/container-entrypoint-with-env

```diff
@@ -1,5 +1,32 @@
 #!/bin/bash
 
+stop_container() {
+    # This script is running as PID=1. Stopping this
+    # process will stop the entire container.
+    exit 0
+}
+trap stop_container TERM INT
```

Comment:
Could you elaborate on why this is required? What does bash do by default if it receives these signals?